docs/pages/includes/node-logins.mdx

```diff
@@ -0,0 +1,30 @@
+When Teleport's Auth Service receives a request to list Teleport Nodes (e.g., to
+display Nodes in the Web UI or via `tsh ls`), it only returns the Nodes that the
+current user is authorized to access.
+
+For each Node in the user's Teleport cluster, the Auth Service applies the
+following checks in order and, if one check fails, hides the Node from the user:
+
+{/*
+
+ TODO: We might want to mention that the Auth Service checks the resource's
+ namespace as well, but we currently do not document resource namespaces.
+ This would not be the appropriate place to include our only namespace
+ documentation. See gravitational/teleport issue #12580.
+ 
+*/}
+
+- None of the user's roles contains a `deny` rule that matches the Node's labels.
```

Comment:
English grammar question: is "contains" the correct form here, or should it say "contain"? :) I'd think that "none of the roles contain" is more correct cause it's plural but I may be mistaken.

Reply:
From what I've read, "none" can be singular or plural, and there's no good rule for this. It probably would sound better as "contain," though.